database/main.go

```diff
@@ -108,7 +127,9 @@ func saveWorkspaceDB(
 	}
 
 	if err := g.WriteFileInWorkspace(ctx, dbWorkspacePath, updatedDBData, gptscript.WriteFileInWorkspaceOptions{
-		WorkspaceID: workspaceID,
+		WorkspaceID:    workspaceID,
+		CreateRevision: &([]bool{true}[0]),
+		LatestRevision: revisionID,
 	}); err != nil {
 		return fmt.Errorf("Error writing updated DB file to workspace: %v", err)
 	}
```

Comment:
Not sure I'm using the feature correctly, but if this is right I'll need to delete the last revision after creating the new one.